pkg/liquidity-mining/test/GaugeMaxRelativeWeight.test.ts

```diff
@@ -0,0 +1,213 @@
+import { ethers } from 'hardhat';
+import { BigNumber, Contract } from 'ethers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { deploy, deployedAt } from '@balancer-labs/v2-helpers/src/contract';
+import { bn, fp } from '@balancer-labs/v2-helpers/src/numbers';
+import Vault from '@balancer-labs/v2-helpers/src/models/vault/Vault';
+import { expect } from 'chai';
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+import { ANY_ADDRESS, ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+describe('GaugeMaxRelativeWeight', () => {
+  let vault: Vault;
+  let gaugeController: Contract;
+  let adaptor: Contract;
+  let admin: SignerWithAddress;
+  let BAL: Contract, token0: Contract;
+
+  let liquidityGaugeImplementation: Contract;
+  let liquidityGaugeFactory: Contract, stakelessGaugeFactory: Contract;
+  let liquidityGauge: Contract, stakelessGauge: Contract;
+
+  let gauge: Contract;
+
+  before('setup signers', async () => {
+    [, admin] = await ethers.getSigners();
+  });
+
+  sharedBeforeEach('deploy authorizer', async () => {
+    vault = await Vault.create({ admin });
+
+    adaptor = await deploy('AuthorizerAdaptor', { args: [vault.address] });
+    gaugeController = await deploy('MockGaugeController', { args: [ZERO_ADDRESS, adaptor.address] });
+
+    await gaugeController.add_type('Ethereum', 0);
```

Comment:
```suggestion
    await gaugeController.add_type('Ethereum', GAUGE_TYPE);
```

Make a constant out of this

Reply:
Done, although it's the type weight, and it's not actually being used in the mock.